experimental/benchmarks/node-dc-eis/fp.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+function getFootprint() {
+if [ -z $1 ]; then
+	PIDS=`ps -ef|grep /node|grep -v grep|grep -v tee|grep -v sh|awk {'print $2'}`
+else
+	PIDS=$1
+fi
+export PIDS
+FP_LOG=$LOGDIR_TEMP/footprint_output.log
+for SERVER_PID in $PIDS
+do
+				FOOTPRINT=`ps -p ${SERVER_PID} -o rss,vsz,comm,pid | tail -n 1 | awk '{ print $1 }'`
```

Comment:
sorry for nit-pick, but formatting of this file is not same as other files.

Reply:
@kunalspathak I compared this to fp.sh in acmeair. It is same except one new line at the end. Which file are you comparing against?

Reply:
I was just comparing against the other bash files but if the formatting is similar to what we have in acmeair, then ignore my comment.

Reply:
@kunalspathak I'd changed it for better by replacing 4-tabs by just one but then I'd reversed changes to match acmeair's fp.sh script. Anyway, is this ready for landing?